### ISSUE TITLE
warning of fail submit

### ISSUE THREAD

**Issue by vlee-harmonicinc** [eric-yhy/teamform-seed#11]
I tried to edit user profile with null description and fail. 
By checking console, it seem that firebase don't support null description. 
It better give user some warning after fail submission or some tip that which field is compulsory, such as '*'. 
angular.js:13708 Error: Firebase.set failed: First argument contains undefined in property 'users.TXjWNCX8GwZ2RMqMl8CFMeRcHPc2.description'
    at Error (native)
    at Bf (http://localhost:8080/lib/firebase.js:335:67)
    at http://localhost:8080/lib/firebase.js:336:166
    at Cb (http://localhost:8080/lib/firebase.js:254:627)
    at Bf (http://localhost:8080/lib/firebase.js:335:424)
    at Af (http://localhost:8080/lib/firebase.js:334:682)
    at U.g.set (http://localhost:8080/lib/firebase.js:462:241)
    at m.$scope.submitUserData (http://localhost:8080/js/profile.js:42:48)
    at fn (eval at compile (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.min.js:231:409), <anonymous>:4:233)
    at b (https://ajax.googleapis.com/ajax/libs/angularjs/1.5.7/angular.min.js:126:19)

**Comment by RonaldShum** [comment]
I have tried to submit "null" description on chrome and success.
Please try submit profile via chrome again.

**Comment by vlee-harmonicinc** [comment]
I tried via chrome...
Maybe it is because the testing account I used was created before update description then.

**Comment by RonaldShum** [comment]
I found the bug. I will fix it in the next pull request.
